tests/php/Security/RememberLoginHashTest.yml

```diff
@@ -0,0 +1,7 @@
+'SilverStripe\Security\Member':
```

Comment:
I'm honestly surprised you used yml fixtures given all your previous DataObject hydration work :-)

Reply:
Old habbits